devblog/2023-10-18-utility-score-bug-post-mortem.mdx

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: MOBILE Utility Score Bug Post Mortem (Pre HIP-88)
+date: 2021-10-18
+hide_table_of_contents: true
+authors: [cokes518]
```

Comment:
Let's add you to the authors table. I can push it to this branch.

Reply:
Oh oops, sorry had copied this from another one